Source/Requests/HARequestType.swift

```diff
@@ -1,9 +1,13 @@
+import Foundation
+
 /// The command to issue
 public enum HARequestType: Hashable, Comparable, ExpressibleByStringLiteral {
     /// Sent over WebSocket, the command of the request
     case webSocket(String)
     /// Sent over REST, the HTTP method to use and the post-`api/` path
     case rest(HAHTTPMethod, String)
+    /// Sent over WebSocket, the binary data to write
+    case data(Data)
```

Comment:
handler id, not data here. The data should be in the 'data' dictionary (which should probably just allValues->map-as-Data->send, or something, for this request type).